cmd/di.go

```diff
@@ -483,13 +485,25 @@ func (di *Dependencies) bootstrapIdentityComponents(options node.Options) {
 
 func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) (err error) {
 	di.IPResolver = ip.NewResolver(options.IPDetectorURL)
-	di.LocationResolver, err = location.CreateLocationResolver(di.IPResolver, options.Country, options.City, options.Type, options.NodeType, options.Address, options.ExternalDb, configDirFlag)
+
+	switch options.Type {
+	case node.LocationTypeManual:
+		di.LocationResolver = location.NewStaticResolver(options.Country, options.City, options.NodeType, di.IPResolver)
+	case node.LocationTypeBuiltin:
+		di.LocationResolver, err = location.NewBuiltInResolver(di.IPResolver)
+	case node.LocationTypeMMDB:
+		di.LocationResolver, err = location.NewExternalDBResolver(filepath.Join(configDirectory, options.Address), di.IPResolver)
```

Comment:
do we really want to use the same param for both oracle address and external db? Looking at `options.Address`

Reply:
Yes, it's more simple and possible to reuse on all providers, it's by DNS convention https://en.wikipedia.org/wiki/Data_source_name

Maybe we should rename it DSN